tests/test_pyi_files.py

```diff
@@ -45,20 +78,34 @@ def test_pyi_file(path: str) -> None:
     run_results = [
         # Passing a file on command line
         subprocess.run(
-            ["flake8", "-j0", *flags, path],
+            [sys.executable, "-Wa", "-m", "flake8", "-j0", *flags, path],
```

Comment:
Does the same thing, but easier to understand and search:
```suggestion
            [sys.executable, "-Walways", "-m", "flake8", "-j0", *flags, path],
```